test/parse/bad-toplevel.txt

```diff
@@ -1,13 +1,7 @@
 ;;; ERROR: 1
 (foo bar)
 (;; STDERR ;;;
-out/test/parse/bad-toplevel.txt:2:2: error: unexpected token "foo"
+out/test/parse/bad-toplevel.txt:2:1: error: unexpected token "(", expected a module field or a command.
```

Comment:
I know this is a first pass conversion (and it's awesome) but this one jumped out at me and confusing. Perhaps when an invalid field is discovered it consumes the Lpar [before erroring](https://github.com/WebAssembly/wabt/pull/591/files#diff-3f63016a605cad7c823d8c04253ed961R682) so the invalid token is given?